### DISCUSSION TITLE
Signing key

### DISCUSSION THREAD

**Discussion by TaliskerMan** [kha7iq/pingme#55] [Q&A]
Is there a signing key for the software, or a has sum to validate the builds? Curious as that is one thing that helps assure valid code.

**Comment by kha7iq** [comment]
Hello @TaliskerMan 
You can find the [checksum](https://github.com/kha7iq/pingme/releases/download/v0.2.3/pingme_checksums.txt) under releases for all the binaries when ever a new version is released.
Hope this helps, let me know if there is anything else.
